corgi/tasks/management/commands/tagcomponent.py

```diff
@@ -0,0 +1,115 @@
+from django.core.management.base import BaseCommand, CommandParser
+
+from corgi.core.models import Component, ComponentTag
+
+
+class Command(BaseCommand):
+
+    help = "Component tag management."
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument(
+            "--purl",
+            dest="component_purl",
+            help="Select component to tag using component PURL",
+        )
+        parser.add_argument(
+            "--name",
+            dest="component_name",
+            help="Select component(s) to tag using component name",
+        )
+        parser.add_argument(
+            "--tag-name",
+            dest="tag_name",
+            help="Set tag name",
+        )
+        parser.add_argument(
+            "--tag-value",
+            dest="tag_value",
+            help="Set tag value",
+        )
+        parser.add_argument(
+            "--operation",
+            dest="operation",
+            help="Set tag operations (get,add,remove)",
+        )
+
+    def handle(self, *args, **options):
+        operation = "get"
+        if options["operation"]:
+            operation = options["operation"]
+        component_name = None
+        if options["component_name"]:
+            component_name = options["component_name"]
+        component_purl = None
+        if options["component_purl"]:
+            component_purl = options["component_purl"]
+        tag_name = None
+        if options["tag_name"]:
+            tag_name = options["tag_name"]
+        tag_value = ""
+        if options["tag_value"]:
+            tag_value = options["tag_value"]
+        self.stdout.write(
+            self.style.SUCCESS(
+                f"operation: {operation}",
+            )
+        )
+        self.stdout.write(
+            self.style.SUCCESS(
+                f"component: {component_name}{component_purl}",
+            )
+        )
+        self.stdout.write(
+            self.style.SUCCESS(
+                f"tag: {tag_name}:{tag_value}",
+            )
+        )
+
+        tag_params = {}
+        if tag_name:
+            tag_params["name"] = tag_name
+        if tag_value:
+            tag_params["value"] = tag_value
```

Comment:
See below comment for explanation:
```suggestion
        if tag_name:
            tag_params["tags__name"] = tag_name
        if tag_value:
            tag_params["tags__value"] = tag_value
```